main/pipeline/pipeline.cc

```diff
@@ -203,7 +442,15 @@ ast::ParsedFile indexOne(const options::Options &opts, core::GlobalState &lgs, c
             if (file.data(lgs).strictLevel == core::StrictLevel::Ignore) {
                 return emptyParsedFile(file);
             }
-            auto parseTree = runParser(lgs, file, print, opts.traceLexer, opts.traceParser);
+
+            unique_ptr<parser::Node> parseTree;
+
+            if (parser == "prism") {
```

Comment:
Could you please switch this to a `switch` (pun intended), and add a `default` case that errors-out?

If we hit that, it indicates that we changed `main/options/options.cc`, but forgot to handle the new value here.

Reply:
So I attempted to implement this because it makes sense, but I realized that Sorbet is implemented in a way that would actually make it impossible to reach the `default` part of the switch statement. I can program the `options.cc` file to recognize if an invalid parser option is passed, then just use the default option (`sorbet`) instead.